.github/workflows/lint.yml

```diff
@@ -8,7 +8,7 @@ run-name: Executing lint QA on ${{ github.repository }} 🚀
 
 on:
   pull_request:
-    branches: [main]
+    branches: [main, main-stbr, main-stbr-tfm-5]
```

Comment:
Just wanted to check that this is intentional and not some temporary code that was intended to be removed before merging this feature branch into main